### ISSUE TITLE
If text color and heading color are set to white in Appearance >> Customize >> Typography, the text is also white when editing pages making the text invisible

### ISSUE THREAD

**Issue by grigorijurasov** [woocommerce/storefront#1181]
**Describe the bug**
If text color and heading color are set to white in Appearance >> Customize >> Typography (that might be necessary if the site background color is black) - the text is also white when editing pages making the text invisible.

**To reproduce**
1. Go to Appearance >> Customize >> Typography 
2. Set the text color and heading color to white and publish.
3. Head back to wp-admin >> Pages and open a random page
4. See the error

**Expected behavior**
I'd expect either the text to remain black on white background in page editor, or to see the background of the same color as the page so that the text is readable/editable.

**Screenshots**

<img width="293" alt="Screenshot 2019-07-11 at 16 23 21" src="https://user-images.githubusercontent.com/13084901/61054951-2980c580-a3f9-11e9-9dc6-2dcab2ebd590.png">

<img width="984" alt="Screenshot 2019-07-11 at 16 23 07" src="https://user-images.githubusercontent.com/13084901/61054960-2dace300-a3f9-11e9-8c58-c99b7051f800.png">

**Desktop (please complete the following information):**
 - OS: MacOs
 - Browser: Chrome
 - Version: 75.0.3770.100

**Additional context**
- WordPress 5.2.2

**Comment by dphyled** [comment]
Another report of this issue in ticket 2300139-zen

**Comment by fragglejack** [comment]
I have this problem also.

**Comment by juliaamosova** [comment]
Thanks for raising this issue @grigorijurasov . I can reproduce it. 

Here is how the post looks at the front end:

![dark](https://user-images.githubusercontent.com/19143190/68949176-c09bc680-07b9-11ea-825e-a45ee66530cb.jpg)

And here is how it looks at the back end of you try to edit it using Gutenberg editor:

![light](https://user-images.githubusercontent.com/19143190/68949183-c42f4d80-07b9-11ea-817c-71464864b34b.jpg)

We should likely keep the text of the page / post black in the editor despite of the color it has at the front end.

**Comment by ccwalburn** [comment]
Another user has reported this issue here: 2537490-zen

**Comment by Robertght** [comment]
This is what I used a temporary workaround through the Code Snippets plugin:

add_action('admin_head', 'my_custom_fonts');

```
function my_custom_fonts() {
  echo '<style>
    .editor-styles-wrapper .editor-block-list__block, .editor-styles-wrapper h1, .editor-styles-wrapper h2, .editor-styles-wrapper h3, .editor-styles-wrapper h4, .editor-styles-wrapper h5, .editor-styles-wrapper h6 {
      color: black;
    } 
  </style>';
}
```

**Comment by swordsmagic** [comment]
why was this removed from the milestone? its still a critical issue.

**Comment by haszari** [comment]
Thanks for keeping an eye on this issue @swordsmagic! We agree that this highlights a significant bug in the theme. However since it only happens in specific cases, and can be worked around in the meantime, we don’t consider it _critical_.

For 2.5.4 we’re focusing on getting open maintenance PRs wrapped up. We may look at addressing other issues in 2.5.4 if we have time. Otherwise we will prioritize issues like this in a future release.

**Comment by Aljullu** [comment]
The fix for this issue will be included in Storefront 2.5.6, which is planned to be released in a couple of weeks. If you want, you can already try the Release Candidate:
https://github.com/woocommerce/storefront/releases/tag/version%2F2.5.6-rc.1
We did a lot of internal testing, but if you find an issue, we would appreciate if you could report it.
